app/views/doc/ext.html.erb

```diff
@@ -228,5 +228,15 @@
         </li>
        </ul>
      </div>
+     <div class='column-left'>
```

Comment:
I think this is not the right approach, have a look at how it is done on the Books section (https://github.com/git/git-scm.com/blob/master/app/views/doc/ext.html.erb#L97) 

You should add your <ul> element, containing the item info around line 218